src/test/java/seedu/clinic/logic/commands/HelpCommandTest.java

```diff
@@ -51,37 +50,28 @@ public void execute_genericHelp_success() {
 
     @Test
     public void execute_addHelp_success() {
-        String generalDescriptionOfAdd = "There are three types of add command to add warehouse,"
-                + " add supplier or add a product to a supplier.";
+        String generalDescriptionOfAdd = "Add\nAdd command allows you to add warehouse or supplier"
```

Comment:
Here too, we should review it if these are retrieved from our individual commands